docs/user/convert/function_usage.rst

```diff
@@ -175,20 +175,47 @@ Usage: ``dateTime($1)``
 
 Example: ``dateTime('2015-01-01T00:00:00.000Z')``
 
-basicDate
-^^^^^^^^^
+basicIsoDate
+^^^^^^^^^^^^
+
+Description: A date format for ``yyyyMMdd``, equivalent to java.time.format.DateTimeFormatter.BASIC_ISO_DATE.
```

Comment:
I wonder if a table of function name, format (yyyyMMdd), example would be helpful....

Reply:
i'll add one

Reply:
You are my hero!